.github/workflows/disable_independent_workflows.sh

```
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Navigate to the workflows directory
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)